serviceAccount.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Hai messo in chiaro questo file, ci sono le tue credenziali firebase. inserisci il file nei gitignore